raster/src/main/scala/geotrellis/raster/ArrayMultiBandTileSubsetMethod.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2016 Azavea.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.raster
+
+
+trait ArrayMultiBandTileSubsetMethod[T <: ArrayMultiBandTile] extends MethodExtensions[T] {
+
+  def subset(m: Map[Int, Int]): ArrayMultiBandTile = {
```

Comment:
I guess that I wanted to treat this more as a permutation with subset being a special case.  I can add these overloads, as well.